webview/src/setup/components/dvc/DvcEnvDetails.tsx

```diff
@@ -14,9 +17,14 @@ export const DvcEnvDetails: React.FC<DvcEnvDetailsProps> = ({
   version,
   isPythonExtensionUsed
 }) => {
-  const versionText = `${
-    version || 'Not found'
-  } (${MIN_CLI_VERSION} <= required < ${MAX_CLI_VERSION}.0.0)`
+  const versionText = (
```

Comment:
Another possibility I thought of was including all info within parenthesis: 

<img width="763" alt="Screenshot 2023-05-09 at 10 46 41 AM" src="https://github.com/iterative/vscode-dvc/assets/43496356/f65dc22e-1d9c-4ee0-9498-836f3551fbb1">

Reply:
Any other ideas for the design?

Reply:
> required up to -> required <> and above?
> tested up to -> tested with

Here's how that looks:

<img width="772" alt="Screenshot 2023-05-09 at 12 04 13 PM" src="https://github.com/iterative/vscode-dvc/assets/43496356/578bb401-8dc1-410e-8397-4d0e1e9faf4b">

Reply:
Can we make it more obvious if the CLI is a minor version ahead of the latest tested version? Maybe add a warning symbol somewhere? The reason is that if there has been a breaking change and the extension no longer works with that version of the CLI then it will not be completely obvious as to why from just looking at the above screen. Maybe PassFilled should become the warning icon? Maybe we should put a warning icon against the version. WDYT?